### PR TITLE
Fixed no such file or directory error in decoratormanager.js Test File

### LINKED ISSUE #836: Fix `no such file or directory error` inside in decoratormanager.js Test File
# Bug Report 🐛

## Expected Behavior

- <h5>Should be able to read files properly </h5>

## Current Behavior

- <h4>When executing the decoratormanager.js test file, an error is encountered: "Error: ENOENT: no such file or directory, open './test/data/decoratorcommands/test.cto'." This indicates that the test file is unable to locate the required test.cto file within the specified directory.</h4>
Screenshot 2024-04-06 at 9 18 16 PM

### Desktop
 - OS: [e.g. macOS]
 - Browser: [e.g. Chrome, Safari]
 - Version: [e.g. 0.22.15]

### PR DESCRIPTION
**Pull request by subhajit20**
# Closes #836 

### Changes

- Added `__dirname` keyword as prefix of every path of the .cto file
- Screenshot 2024-04-06 at 9 26 05 PM

### Author Checklist
- [X] Ensure you provide a [DCO sign-off](https://github.com/probot/dco#how-it-works) for your commits using the `--signoff` option of git commit.
- [ ] Vital features and changes captured in unit and/or integration tests
- [ ] Commits messages follow [AP format](https://github.com/accordproject/techdocs/blob/master/DEVELOPERS.md#commit-message-format)
- [ ] Extend the documentation, if necessary
- [ ] Merging to `main` from `fork:branchname`

### CODE REVIEW

**Comment by subhajit20** [comment]
Hey @DianaLease can you please review this pr?

**Comment by DianaLease** [comment]
What OS are you using?

File: packages/concerto-core/test/decoratormanager.js [hunk 1]
```diff
@@ -62,16 +63,16 @@ describe('DecoratorManager', () => {
 
     describe('#validate', function() {
         it('should support syntax validation', async function() {
-            const dcs = fs.readFileSync('./test/data/decoratorcommands/web.json', 'utf-8');
+            const dcs = fs.readFileSync(path.join(__dirname+'/data/decoratorcommands/web.json'), 'utf-8');
```
Comment (DianaLease) [inline packages/concerto-core/test/decoratormanager.js:66]: Arguments to path.join should be a comma separated list of strings: https://nodejs.org/api/path.html#pathjoinpaths

**Comment by subhajit20** [comment]
> What OS and Node version are you using? Any other specific details about your environment? Wondering why this is failing for you.

I am using Mac OS and node version is `v18.3.0`

File: packages/concerto-core/test/decoratormanager.js (hunk 1 above)
Comment (subhajit20) [reply packages/concerto-core/test/decoratormanager.js]: Yeah!
Fixed that.

**Comment by DianaLease** [comment]
> > What OS and Node version are you using? Any other specific details about your environment? Wondering why this is failing for you.
> 
> I am using Mac OS and my node version is `v18.3.0`. When I execute the test file from the root of the repo then I get this error but if I specifically go inside `concerto-core` folder and try to execute then file the it worked fine. I used `npx mocha` command to run the file and there are other test files where I have faced similar issue as well

I see. So with this change, will will it work from both the root and from concerto-core? @subhajit20

**Comment by subhajit20** [comment]
> > > What OS and Node version are you using? Any other specific details about your environment? Wondering why this is failing for you.
> > 
> > 
> > I am using Mac OS and my node version is `v18.3.0`. When I execute the test file from the root of the repo then I get this error but if I specifically go inside `concerto-core` folder and try to execute then file the it worked fine. I used `npx mocha` command to run the file and there are other test files where I have faced similar issue as well
> 
> I see. So with this change, will will it work from both the root and from concerto-core? @subhajit20

Yup! It is working fine right now after making these changes done both from the root and the `concerto-core` or any other packages Directory

File: packages/concerto-core/test/decoratormanager.js [hunk 2]
```diff
@@ -26,6 +27,7 @@ chai.use(require('chai-things'));
 chai.use(require('chai-as-promised'));
 
 describe('DecoratorManager', () => {
+    console.log(path.join(__dirname,"/data"))
```
Comment (mttrbrts) [inline packages/concerto-core/test/decoratormanager.js:30]:
```suggestion
```

**Pull request merged**